R/paths.R

```diff
@@ -31,12 +36,35 @@
 #'   Default is `NULL`.
 #' @param spread the width of the fan spread
 #'
-#' @return a `tidy_dagitty` with a `path` column for path variables
-#'   and a `set` grouping column or a `ggplot`
-#' @export
+#' @return
+#' \describe{
+#'   \item{`dag_paths`}{A `tidy_dagitty` object with the following additional
+#'   columns in the `data` element:
+#'   \describe{
+#'     \item{`set`}{A character vector representing an id for the identified
+#'     open path.}
+#'     \item{`path`}{A character vector with value `"open path"` if the
+#'     corresponding node is on the identified open path and value `NA` if the
+#'     corresponding node is not on the identified open path.
+#'     If `paths_only` is set to `TRUE` all rows where `path` is `NA` are
+#'     removed.}
+#'   }
+#'   The returned `tidy_dagitty` object duplicates for each identified open path
+#'   all node rows in the `data` element and labels for each identified open
+#'   path (as indicated by column `set`) which nodes of the graph are on the
+#'   path (as indicated by column `path`).
+#'   }
+#'   \item{`ggdag_paths`}{A `ggplot` object with a facet for each identified
+#'   open path.}
+#'   \item{`ggdag_paths_fan`}{A `ggplot` object with the same contents as
+#'   with `ggdag_paths`, but identified open paths are plotted in one panel.}
+#' }
```

Comment:
This is too much detail for my taste, and I'd prefer not to use `\describe{}` and other non-markdown elements unless necessary. Could you revert this to the previous `@return`?

Reply:
Ok, I see that this might be too much and I changed this back as you requested. However, I think that it is not clear what "path variables" are, or what the "`set` grouping column" should represent.